src/api/helpers/get-countries/index.test.ts

```diff
@@ -0,0 +1,31 @@
+import { Context } from '.keystone/types'; // eslint-disable-line
+import getCountries from '.';
+import getKeystoneContext from '../../test-helpers/get-keystone-context';
+
+describe('helpers/get-countries', () => {
+  let context: Context;
+
+  beforeAll(async () => {
+    context = getKeystoneContext();
+  });
+
+  it('should return countries', async () => {
+    const result = await getCountries(context);
+
+    const expectedCountries = await context.db.Country.findMany();
+
+    expect(result).toEqual(expectedCountries);
+  });
+
+  describe('when there is an error', () => {
+    it('should throw an error', async () => {
+      try {
```

Comment:
can we use .toThrow()

Reply:
Tried this before, the error assertion fails. Not sure why this happens, we have this in 44 places.